app/routes/keyword_type.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+const keywordTypeController = require('../controllers/keyword_type');
+
+const express = require('express');
+let router = express.Router();
+
+
```

Comment:
This space here needs to have the keyword route defined. Look at how we do it with users to see what it should look like.

Also, you haven't modified the file yet, but this file needs to be referenced by api.js. Follow the pattern there.